ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -168,8 +168,85 @@ SettingsContentBase {
         }
 
         Rectangle {
-            id: modeSeparator
+            id: scrollSeparator
             anchors.top: zoomSlider.bottom
+            anchors.topMargin: Style.current.padding * 3
+            anchors.left: parent.left
+            anchors.right: parent.right
+            height: 1
+            color: Style.current.separator
+        }
+
+        StatusSectionHeadline {
+            id: labelScrolling
+            anchors.top: scrollSeparator.bottom
+            anchors.topMargin: Style.current.bigPadding * 2
+            anchors.left: parent.left
+            anchors.right: parent.right
+            text: qsTr("Scrolling")
+        }
+
+        StatusBaseText {
+            id: labelScrollVelocity
+
+            anchors.top: labelScrolling.bottom
+            anchors.topMargin: Style.current.padding
+            anchors.left: parent.left
+            color: Style.current.secondaryText
+            font.pixelSize: 13
```

Comment:
```suggestion
            font.pixelSize: Style.current.secondaryTextFontSize
```
I know `secondaryTextFontSize` is 14 not 13 but maybe it would fit.